Makefile

```diff
@@ -116,7 +120,18 @@ gen-api-fail-if-dirty: gen-api
 	)
 .PHONY: gen-api-fail-if-dirty
 
-gen-olm: $(OPERATOR_SDK) gen
+build-operator-registry: $(OPERATOR_SDK) gen
+	@echo "Building image for operator-registry"
+	@rm -rf operator-registry
+	git clone $(OPERATOR_REGISTRY_REPO_URL).git
+	@cd operator-registry; \
+	git -c advice.detachedHead=false checkout tags/$(OPERATOR_REGISTRY_VERSION); \
+	docker build -t quay.io/operator-framework/upstream-registry-builder:$(OPERATOR_REGISTRY_VERSION) -f upstream-builder.Dockerfile .
+	@echo "✅ build-operator-registry"
+.PHONY: build-operator-registry
+
+
+gen-olm: build-operator-registry
```

Comment:
Let's remove this per @guymguym suggestion.

Reply:
Addressed in 8857002